code/modules/projectiles/targeting.dm

```diff
@@ -45,7 +45,7 @@
 /obj/item/weapon/gun/proc/PreFire(atom/A as mob|obj|turf|area, mob/living/user as mob|obj, params, struggle = 0)
 	//Lets not spam it.
 	if(lock_time > world.time - 2) return
-	.
+	. //the rare and mystical rogue ".". nobody knows what this majestic beast does or why is it here. not even why it compiles is known for certain. legend says should it be touched, everything will probably break; but nobody dares try.
```

Comment:
honk

Come on, just remove it
It's just a variable sitting there, doing nothing